app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -33,6 +35,7 @@ type Props = {
     currentUserId: string;
     tutorialWatched: boolean;
     teammateDisplayNameSetting: string;
+    channelAbacPolicyEnforced?: boolean;
```

Comment:
If you make the observable default to false, this should not be optional

Reply:
ok, done in my last commit.